substrafl/remote/register/register.py

```diff
@@ -29,14 +29,30 @@
 MINIMAL_PYTHON_VERSION = 9  # 3.9
 MAXIMAL_PYTHON_VERSION = 12  # 3.12
 
-_DEFAULT_BASE_DOCKER_IMAGE = "python:{python_version}-slim"
+_CPU_BASE_IMAGE = """
+FROM python:{python_version}-slim
 
-DOCKERFILE_TEMPLATE = """
-FROM {docker_image}
+# update image
+RUN apt-get update -y
+"""
+
+_GPU_BASE_IMAGE = """
```

Comment:
Do we want to do something about non-root users in GPU-friendly containers?

Reply:
The user management part has been factorized in the `DOCKERFILE_TEMPLATE` (lines 56 to 61).